src/bootstrap/bin/rustc.rs

```diff
@@ -38,6 +38,18 @@ use std::str::FromStr;
 use std::time::Instant;
 
 fn main() {
+    #[cfg(windows)]
+    unsafe {
+        const SEM_NOGPFAULTERRORBOX: u32 = 0x0002;
+        extern "system" {
+            fn SetErrorMode(mode: u32) -> u32;
+        }
+        // Enable the Windows Error Reporting dialog which bootstrap disabled,
+        // so we can JIT debug rustc
+        let mode = SetErrorMode(0);
+        SetErrorMode(mode & !SEM_NOGPFAULTERRORBOX);
```

Comment:
This seems like it's not necessarily the most appropriate place for this, should the block in `src/bootstrap/job.rs` instead be located in compiletest?

Reply:
That may be better. This code could be moved to `src/bootstrap/job.rs` and then `compiletest` and maybe `rustdoc` would disable Windows Error Reporting again when running tests.